### ISSUE TITLE
Possible Bug in Centre Model being Usable via View Model

### ISSUE THREAD

**Issue by samuelsenior** [Spyderisk/system-modeller#150]
Discovered through issue #149, if a system model is being viewed in the View Model mode then it should not be possible for the user to edit aspects of the model, however, the user is still able to use the Centre Model button. This seems like a possible bug as this button modifies the positions of assets in the system model, and so maybe this option should not be present or should be disabled when in the View Mode?

**Comment by scp93ch** [comment]
Yes, you're correct. The "Centre Model" menu item moves the assets so that the average asset position is in the middle of the canvas. That updates the UI graph of the model and so should not be available in the View mode.
